
arbitrage_old.erl:51: Call to missing or unexported function channel_manager:id/1
arbitrage_old.erl:79: Call to missing or unexported function channel_manager:id/1
arbitrage_old.erl:81: Call to missing or unexported function channel_manager:read_channel/1
arbitrage_old.erl:114: Call to missing or unexported function channel_manager:id/1
arbitrage_old.erl:116: Call to missing or unexported function channel_manager:read_channel/1
channel_feeder.erl:239: Call to missing or unexported function block:channels/1
channel_feeder.erl:241: Function depth_check2/3 has no local return
channel_feeder.erl:247: The call channels:entropy(Channel::{_,'empty' | {'channel',non_neg_integer(),non_neg_integer(),non_neg_integer(),non_neg_integer(),non_neg_integer(),integer(),non_neg_integer(),non_neg_integer(),non_neg_integer(),char(),non_neg_integer(),0,boolean(),_},_}) will never return since it differs in the 1st argument from the success typing arguments: ({'channel',_,_,_,_,_,_,_,_,_,_,_,_,_,_})
channel_feeder.erl:248: The call channels:entropy(OldChannel::{_,'empty' | {'channel',non_neg_integer(),non_neg_integer(),non_neg_integer(),non_neg_integer(),non_neg_integer(),integer(),non_neg_integer(),non_neg_integer(),non_neg_integer(),char(),non_neg_integer(),0,boolean(),_},_}) will never return since it differs in the 1st argument from the success typing arguments: ({'channel',_,_,_,_,_,_,_,_,_,_,_,_,_,_})
channel_manager.erl:45: Call to missing or unexported function dict:delete/2
base58.erl:201: The pattern 'error' can never match the type ['error' | 1..255]
mine.erl:13: The pattern 'true' can never match the type 'false'
secrets.erl:51: Function old_new_lightning/0 will never be called
secrets.erl:57: Function test_new_lightning/0 will never be called
testnet_sup.erl:14: Function child_maker/1 will never be called
testnet_sup.erl:25: Function tree_child/3 will never be called
testnet_sup.erl:27: Function tree_child/4 will never be called
testnet_sup.erl:31: Function init/1 has no local return
testnet_sup.erl:32: The call os:putenv("ERL_CRASH_DUMP_SECONDS",<<_:8>>) breaks the contract (VarName,Value) -> 'true' when VarName :: string(), Value :: string()
accounts.erl:137: Call to missing or unexported function trees:garbage/1
governance.erl:18: The pattern 'true' can never match the type 'false'
orders.erl:149: Guard test #order{id::non_neg_integer(),aid::non_neg_integer(),amount::non_neg_integer()} =:= N::non_neg_integer() can never succeed
spk.erl:255: Clause guard cannot succeed.
testnet_sign.erl:189: Guard test Verbose::'false' =:= 'true' can never succeed
dice.erl:17: Call to missing or unexported function channel_feeder:make_simplification/3
dice.erl:20: Call to missing or unexported function channel_feeder:update_to_me/1
dice.erl:27: Call to missing or unexported function secrets:new/0
dice.erl:37: Call to missing or unexported function channel_feeder:agree_simplification/3
easy.erl:255: Call to missing or unexported function spk:run/7
easy.erl:301: Call to missing or unexported function channel_team_close_tx:make/4
easy.erl:310: Call to missing or unexported function channel_solo_close:make/6
easy.erl:315: Call to missing or unexported function channel_timeout_tx:make/4
easy.erl:320: Call to missing or unexported function channel_slash_tx:make/6
easy.erl:346: Call to missing or unexported function governance:name2number/1
easy.erl:424: Function off/0 has no local return
mail.erl:103: Call to missing or unexported function channel_manager:id/1
mail.erl:121: Call to missing or unexported function create_account_tx:create_account/3
mail.erl:122: Call to missing or unexported function spend_tx:spend/4
mail.erl:139: Call to missing or unexported function tx_pool:accounts/0
mail.erl:141: Call to missing or unexported function keys:sign/1
peers.erl:16: Call to missing or unexported function dict:remove/2
oracle_questions.erl:13: Cons will produce an improper list since its 2nd argument is 'ok' | nonempty_improper_list(any(),'ok')
oracle_questions.erl:24: Function save/1 has no local return
oracle_questions.erl:28: The call block:hash(Binary::binary()) will never return since it differs in the 1st argument from the success typing arguments: ({'block_plus',{'block_plus',{'block_plus',{_,_,_,_,_,_} | {_,_,_,_,_,_,_,_,_,_},_,_,_,_} | {'block',_,_,_,_,_,_,_,_,_},_,_,_,_} | {'block',_,_,_,_,_,_,_,_,_},_,_,_,_} | {'block',_,_,_,_,_,_,_,_,_})
Unknown functions:
  block_tree:account/1
  block_tree:buy_block/0
  chalang:data_maker/7
  chalang:new_state/3
  chalang:none_of/1
  chalang:run5/2
  chalang:stack/1
  chalang:time_gas/1
  chalang:vm/6
  channel_block_tx:acc1/1
  channel_block_tx:acc2/1
  channel_block_tx:bet_amount/1
  channel_block_tx:bet_code/1
  channel_block_tx:bet_to/1
  channel_block_tx:bets/1
  channel_manager_feeder:recieve/3
  channel_manager_feeder:spend_account/2
  channel_timeout:make/4
  compiler:compile/1
  compiler_chalang:doit/1
  cowboy:start_http/4
  cowboy_req:body/1
  cowboy_req:path/1
  cowboy_req:peer/1
  cowboy_req:reply/4
  cowboy_router:compile/1
  crypto:compute_key/4
  crypto:ec_curve/1
  crypto:generate_key/2
  crypto:sign/4
  crypto:strong_rand_bytes/1
  crypto:verify/5
  db:read/1
  db:save/2
  db:test/0
  disassembler:doit/1
  encryption:decrypt/2
  encryption:encrypt/2
  encryption:get_msg/1
  encryption:get_msg/2
  encryption:id/1
  encryption:msg/1
  encryption:send_msg/4
  encryption:test/0
  free_variables:bets/0
  hash:doit/2
  hash:file/1
  httpc:request/4
  inets:start/0
  jiffy:decode/1
  jiffy:encode/1
  leaf:meta/1
  leaf:value/1
  pow:above_min/3
  pow:check_pow/2
  pow:data/1
  pow:pow/4
  pow:recalculate/3
  pow:sci2int/1
  sign_tx:sign/1
  to_channel_tx:create_channel/5
  trie:delete/3
  trie:garbage/2
  trie:get/3
  trie:get_all/2
  trie:put/5
  trie:root_hash/2

  Donate to .. and support Zwilla: 

  * btc:1DaGkc1Uv4GeCSpjkrMVzCA35ENmrd526V 

  * eth:0x284DbB6139e2e08cd3D3BE6f51306c19cAB04e3c 

  * ltc:LNvu63U68G72KXHWP5yqSuSYpoa7ef58c7 

  
